crates/teloxide-core/src/types/update.rs

```diff
@@ -108,6 +127,14 @@ pub enum UpdateKind {
     /// updates.
     ChatJoinRequest(ChatJoinRequest),
 
+    /// A chat boost was added or changed. The bot must be an administrator in
+    /// the chat to receive these updates.
+    ChatBoost(ChatBoostUpdated),
```

Comment:
> the variant should probably be named `ChatBoostUpdated` too?

This could be done, but [TBA serializes](https://core.telegram.org/bots/api#update) this update kind as `chat_member` both in `Update` and `allowed_updates`. So I used here naming from TBA docs. Btw, even though `UpdateKind::ChatBoost` can be renamed to `ChatBoostUpdated`, `AllowedUpdate::ChatBoost` should stay as is because `AllowedUpdate::ChatBoostUpdated` doesn't work.